Sources/main.cpp

```
@@ -99,6 +101,12 @@ static int PlatformMain(std::string const &host, std::string const &port) {
 
   std::unique_ptr<Socket> serverSocket = CreateSocket(host, port, false);
 
+#if defined(OS_POSIX)
```

What's the point? I'd rather fail early at build time than at run time.